spyderlib/qt/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 import os
 
-os.environ.setdefault('QT_API', 'pyqt')
+os.environ.setdefault('QT_API', 'pyqt5')
```

Comment:
> there are many bugs that need to be fixed

as said before: please point them out to me.

> so there is no pressing need to have this merged

makes everything faster, smoother and better looking on systems where PyQt5 is supported, poses no problems on other systems. reason enough?

> When the distributions provides such packages and we can reliably test how qt5 and pyqt5 works in all the different OS we can then go forward on this. 

and until then nobody can use it, so we don’t need to worry